common/git-common/src/test/java/org/arquillian/pact/common/git/GitOperationsTest.java

```diff
@@ -0,0 +1,82 @@
+package org.arquillian.pact.common.git;
+
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.jgit.lib.Ref;
+import org.eclipse.jgit.revwalk.RevCommit;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GitOperationsTest {
+
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    @Test
+    public void should_check_if_branch_is_local() throws IOException {
+        GitOperations gitOperations = new GitOperations();
+        Git git = gitOperations.cloneRepository("https://github.com/github/testrepo.git",
```

Comment:
I thought the same, but I followed the same approach used in jgit. The good thing about this repo is that it contains a bit of everything, commits, branches, tags, ... so if we do it by ourself, we will need to provide everything.  Since this test is not really critical, I mean it just checks that we are configuring jgit correctly, something that we are not going to touch often, maybe I will put an assumeThat